### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by h9419**
Just updating docs to manage expectation on iOS 15.4-15.7 rootless jailbreak support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <p>palera1n</p>
 </h1>
 <h3 align="center">An iOS 15.0-15.3.1 work-in-progress, tethered checkm8 jailbreak.</h3>
+<h3 align="center">iOS 15.0-15.7.1 work-in-progress, tethered rootless checkm8 jailbreak.</h3>
```
Comment (itsnebulalol) [inline README.md:6]: Not a big fan of this part... any way to improve this?

Comment (h9419) [reply README.md:6]: Sure, I can remove this line. The body already clarified which version is supported

Comment (h9419) [reply README.md]: Removed the second subheading

**Comment by Unknowndude81** [comment]
I'm waiting for it. I've an A10 with iOS 15.7.1 to test new code on.

**Comment by PaddyPan520** [comment]
My iPhoneX, 15.6. encountered disk write issues when using the latest tools. 'Darwin/gtar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.quarantine'
hdiutil: couldn't eject "disk3" ’

**Comment by PaddyPan520** [comment]
> 我的 iPhoneX，15.6。使用最新工具时遇到磁盘写入问题。'Darwin/gtar：忽略未知的扩展标题关键字' LIBARCHIVE.xattr.com.apple.quarantine'hdiutil：无法弹出“disk3”'

14.8 iPhone7plus too

**Comment by AldeabAhmed** [comment]
> My iPhoneX, 15.6. encountered disk write issues when using the latest tools. 'Darwin/gtar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.quarantine' hdiutil: couldn't eject "disk3" ’

restart you PC

**Comment by AldeabAhmed** [comment]
> My iPhoneX, 15.6. Down all the step and I can see 'Enjoy!' but my phone .
> 
> but the phone has a black screen, and the connection to the computer doesn't respond like it's turned off. All we do is force a reboot. can someone help me

reclone into a new path and in the terminal write this:
./palera1n.sh --dfu 15.6

knowing that still rootless

**Comment by itsnebulalol** [comment]
This is *not* the place for this. Join our discord for help.

**Pull request closed without merging**